test/code/providers/runas_provider/runasprovider_test.cpp

```diff
@@ -109,8 +122,14 @@ class SCXRunAsProviderTest : public CPPUNIT_NS::TestFixture
         TestableContext context;
         TearDownAgent<mi::SCX_OperatingSystem_Class_Provider>(context, CALL_LOCATION(errMsg));
         CPPUNIT_ASSERT_EQUAL_MESSAGE(ERROR_MESSAGE, false, context.WasRefuseUnloadCalled() );
-        system("rm -rf testChRoot > /dev/null 2>&1");
-        system("rm -rf testCWD > /dev/null 2>&1");
+        int err = system("rm -rf testChRoot > /dev/null 2>&1") | system("rm -rf testCWD > /dev/null 2>&1") | 
+        		system("rm ./tmpdir > /dev/null 2>&1");
+        if(err) 
+        { 
+        	std::ostringstream os;
+            os << "Unexpected error in tearDown: " << SCXCoreLib::strerror(err & 127);
+            std::cout<<os.str()<<std::endl;
+        }
```

Comment:
See comments above. If you keep this in tearDown, at least use SCXProcess::Run so you could dump stdout/stderr in the error case.